Buckify Sigmoid test

3l1 opened the pull request:
Summary: Buckify Sigmoid test

Reviewed By: digantdesai

Differential Revision: D72940392

3l1 commented:
@pytorchbot label "topic: not user facing"

digantdesai commented on backends/arm/test/ops/test_sigmoid.py:
```diff
@@ -145,35 +154,44 @@ def test_sigmoid_tosa_MI(
         )
 
     @parameterized.expand(test_data_suite)
+    @pytest.mark.tosa_ref_model
```
these are not running on tosa_ref_model

digantdesai commented on backends/arm/test/ops/test_sigmoid.py:
```diff
@@ -122,6 +128,9 @@ def _test_sigmoid_tosa_ethos_BI_pipeline(
             .to_executorch()
         )
 
+        if conftest.is_option_enabled("tosa_ref_model"):
+            tester.run_method_and_compare_outputs(inputs=test_data)
```
This is not right, we don't want to change the way we do things on U55/U85 just for tosa-ref-model.

3l1 committed:
Buckify Sigmoid test (#10224)

Summary:
Pull Request resolved: https://github.com/pytorch/executorch/pull/10224

Buckify Sigmoid test

Reviewed By: digantdesai

Differential Revision: D72940392
